feature-configs/demo/sctp/sctp_machineconfigpool_patch.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfigPool
+metadata:
+  name: sctp-pool
+spec:
+  nodeSelector:
+    matchLabels:
+      node-role.kubernetes.io/worker-sctp: ""
```

Comment:
I think this is the same as you have in base already? In that case, we don't need a patch here

Reply:
Will add something different just to show how kustomize work.

Reply:
done